debian/control

```diff
@@ -2,7 +2,7 @@ Source: debathena-pyhesiodfs
 Section: debathena/net
 Priority: extra
 Maintainer: Debathena Project <debathena@mit.edu>
-Build-Depends: cdbs, debhelper, config-package-dev, python-support, python (>= 2.6~)
+Build-Depends: cdbs, debhelper, dh-systemd (>= 1.5), config-package-dev, python-support, python (>= 2.6~)
```

Comment:
Debathena builds simultaneously for all supported releases without branching, as this greatly simplifies the build infrastructure and release management process. One could imagine reasons to potentially make an exception, but this is not likely to be one of them, since there are easy workarounds, such as `dh-systemd (>= 1.5) | cdbs (<< 0.4.122~)`.